lib/core/kd.coffee

```diff
@@ -44,19 +46,14 @@ module.exports =
   deleteInstance: (id) ->
     delete instances[id]
 
+  deleteInstances: -> instances = []
```

Comment:
you probably meant `instances = {}` ?
https://github.com/koding/kd/pull/163/files#diff-2e2f9603b796166555ec8807de005c9bR5